WordPress/Classes/Utility/HelpshiftUtils.m

```diff
@@ -107,6 +109,20 @@ + (void)refreshUnreadNotificationCount
     [HelpshiftSupport getNotificationCountFromRemote:YES];
 }
 
++ (NSArray<NSString *> *)planTagsForAccount:(WPAccount *)account
+{
+    NSMutableSet<NSString *> *tags = [NSMutableSet set];
+    for (Blog *blog in account.blogs) {
+        if (blog.planID == nil) {
+            continue;
+        }
+        NSString *tag = [NSString stringWithFormat:@"plan:%@", blog.planID];
+        [tags addObject:tag];
+    }
+
+    return [tags allObjects];
```

Comment:
Is there a reason you're no longer removing the Free tag? (you were doing so in the original version of this PR)